### ISSUE TITLE
merge double-carriageway roads on low zoom level tiles

### ISSUE THREAD

**Issue by miyuer** [openmaptiles/openmaptiles#380]
how does merge double-carriageway roads on low zoom level tiles.

**Comment by jirik** [comment]
Good idea. It should be possible, but probably not so easy. Merging it on lower zoom levels means to do some geometry processing that may affect performance a lot.

We can also use [dual_carriageway](https://wiki.openstreetmap.org/wiki/Relation:dual_carriageway) relation (not imported right now).

@miyuer Feel free to make some tests and -- if it looks promising -- also PR.

**Comment by ZeLonewolf** [comment]
Related to #1150
